packages/flutter_tools/test/commands/create_test.dart

```diff
@@ -299,6 +302,23 @@ void main() {
         throwsToolExit(message: '"invalidName" is not a valid Dart package name.'),
       );
     });
+
+    testUsingContext('invokes pub offline', () async {
+      Cache.flutterRoot = '../..';
+
+      final CreateCommand command = new CreateCommand();
+      final CommandRunner<Null> runner = createTestCommandRunner(command);
+
+      await runner.run(<String>['create', '--pub', projectDir.path]);
+      final List<String> commands = loggingProcessManager.commands;
+      expect(commands, contains(matches(r'dart-sdk[\\/]bin[\\/]pub')));
+      expect(commands, contains('--offline'));
```

Comment:
this should fail now, no?

Reply:
No.  It's testing what the command line sent to `pub` is, and so it should now always include `--offline`, since I turned that on by default.